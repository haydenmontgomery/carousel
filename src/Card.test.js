import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import photos from "./photos.js";
import TEST_IMAGES from "./_testCommon.js";

test('it renders without crashing', () => {
  render(<Card caption="Test"
    src={photos[0].src}
    currNum={0}
    totalNum={0}/>)
});

test("it matches snapshot", () => {
  const { asFragment } = render(<Card caption="Test"
    src={photos[0].src}
    currNum={0}
    totalNum={0}/>)
  expect(asFragment()).toMatchSnapshot();
})