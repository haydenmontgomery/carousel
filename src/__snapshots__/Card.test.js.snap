// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Test
    </h4>
    <img
      alt="Test"
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 0 of 0.
    </small>
  </div>
</DocumentFragment>
`;
